worlds/musedash/MuseDashData.py

```diff
@@ -0,0 +1,611 @@
+    "Euphoria": SongData(2900661, "71-5", "Valentine Stage", True, 1, 3, 6),
+    "P E R O P E R O Brother Dance": SongData(2900663, "72-0", "Legends of Muse Warriors", True, None, 7, None),
+    "PA PPA PANIC": SongData(2900664, "72-1", "Legends of Muse Warriors", False, 4, 8, 10),
+    "How To Make Music Game Song!": SongData(2900665, "72-2", "Legends of Muse Warriors", True, 6, 8, 10),
+    "Re Re": SongData(2900666, "72-3", "Legends of Muse Warriors", True, 7, 9, 11),
+    "Marmalade Twins": SongData(2900667, "72-4", "Legends of Muse Warriors", True, 5, 8, 10),
+    "DOMINATOR": SongData(2900668, "72-5", "Legends of Muse Warriors", True, 7, 9, 11),
+    "Teshikani TESHiKANi": SongData(2900669, "72-6", "Legends of Muse Warriors", True, 5, 7, 9),
+    "Urban Magic": SongData(2900671, "73-0", "Happy Otaku Pack Vol.19", True, 3, 5, 7),
+    "Maid's Prank": SongData(2900672, "73-1", "Happy Otaku Pack Vol.19", True, 5, 7, 10),
+    "Dance Dance Good Night Dance": SongData(2900673, "73-2", "Happy Otaku Pack Vol.19", True, 2, 4, 7),
+    "Ops Limone": SongData(2900674, "73-3", "Happy Otaku Pack Vol.19", True, 5, 8, 11),
+    "NOVA": SongData(2900675, "73-4", "Happy Otaku Pack Vol.19", True, 6, 8, 10),
+    "Heaven's Gradius": SongData(2900676, "73-5", "Happy Otaku Pack Vol.19", True, 6, 8, 10),
+    "Ray Tuning": SongData(2900678, "74-0", "CHUNITHM COURSE MUSE", True, 6, 8, 10),
+    "World Vanquisher": SongData(2900679, "74-1", "CHUNITHM COURSE MUSE", True, 6, 8, 10),
+    "Tsukuyomi Ni Naru": SongData(2900680, "74-2", "CHUNITHM COURSE MUSE", True, 5, 7, 9),
+    "The wheel to the right": SongData(2900681, "74-3", "CHUNITHM COURSE MUSE", True, 5, 7, 9),
+    "Climax": SongData(2900682, "74-4", "CHUNITHM COURSE MUSE", True, 4, 8, 11),
+    "Spider's Thread": SongData(2900683, "74-5", "CHUNITHM COURSE MUSE", True, 5, 8, 10),
+    "HIT ME UP": SongData(2900684, "43-54", "MD Plus Project", True, 4, 6, 8),
+    "Test Me feat. Uyeon": SongData(2900685, "43-55", "MD Plus Project", True, 3, 5, 9),
+    "Assault TAXI": SongData(2900686, "43-56", "MD Plus Project", True, 4, 7, 10),
+    "No": SongData(2900687, "43-57", "MD Plus Project", False, 4, 6, 9),
+    "Pop it": SongData(2900688, "43-58", "MD Plus Project", True, 1, 3, 6),
+    "HEARTBEAT! KyunKyun!": SongData(2900689, "43-59", "MD Plus Project", True, 4, 6, 9),
+    "SUPERHERO": SongData(2900691, "75-0", "Novice Rider Pack", False, 2, 4, 7),
+    "Highway_Summer": SongData(2900692, "75-1", "Novice Rider Pack", True, 2, 4, 6),
+    "Mx. Black Box": SongData(2900693, "75-2", "Novice Rider Pack", True, 5, 7, 9),
+    "Sweet Encounter": SongData(2900694, "75-3", "Novice Rider Pack", True, 2, 4, 7),
+    "Echo over you... Secret": SongData(2900695, "0-55", "Default Music", False, 6, 8, 10),
+    "Echo over you...": SongData(2900696, "0-56", "Default Music", False, 1, 4, None),
+    "Tsukuyomi Ni Naru": SongData(2900697, "74-6", "CHUNITHM COURSE MUSE", True, 5, 8, 10),
```

Comment:
Just commenting with the the two Tsukuyomi Ni Naru lines at the top and bottom here

Reply:
Error on the generation side that kept replacing the specific name with the wrong one. Resolved and pushed.